python/foglamp/common/configuration_manager.py

```diff
@@ -601,13 +601,16 @@ def get_entry_val(k):
         """
         try:
             if category_name in self._cacheManager:
-                return self._cacheManager.cache[category_name]['value']
+                # Interim solution; to ensure script type config item file content handling
+                category_value = self._handle_script_type(category_name, self._cacheManager.cache[category_name]['value'])
+                self._cacheManager.update(category_name, category_value, display_name=self._cacheManager.cache[category_name]['displayName'])
+                return category_value
 
             category_value = await self._read_category_val(category_name)
 
             if category_value is not None:
-                self._cacheManager.update(category_name, category_value)
                 category_value = self._handle_script_type(category_name, category_value)
+                self._cacheManager.update(category_name, category_value)
```

Comment:
this will force here to set `display_name = category_name`, we should test that and create a ticket to check `                 self._cacheManager.update(category_name, category_value)` calls and handle this; And also a story to handle category-> description as well.

Reply:
FOGL-3246